index.html

```diff
@@ -0,0 +1,157 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<title>html and css cards quiz</title>
+	<link rel="stylesheet" href="quiz.css">
+  <style type="quiz.css" media="screen">
+
+  </style>
+</head>
+<body>
+
+<article id="cardOne">
```

Comment:
The article tag should be indented inside the body tag, same for the other card articles